index.js

```diff
@@ -0,0 +1,19 @@
+// Первый аргумент - массив, второй - колбек-функция, которая применяется на массив
+// Колбек-функция применяется на каждый элемент массива и проверяет условие
+// Если хотя бы один раз колбек-функция вернула true, то any возвращает true
+// Иначе - false
+// Если колбек-функция не была передана, то из any возвращаем true, 
+// если в массиве есть хотя бы один элемент, который дает true,
+// если все значения в массиве дают false, то возвращаем false
+
+//O(n)
+function any(array, collback = (x) => !!x === true) {
+  return array.filter(collback).length > 0;
```

Comment:
Try to find more suitable method here

Reply:
Now I use some() method, because if one matching element is found, it will immediately return true before reaching the end of the array